h2/src/test/org/h2/test/TestBase.java

```diff
@@ -167,8 +166,15 @@ public void runTest(TestAll conf) {
      * @return the connection
      */
     public Connection getConnection(String name) throws SQLException {
-        return getConnectionInternal(getURL(name, true), getUser(),
-                getPassword());
+        return getConnection(name, null);
+    }
+
+    public Connection getConnection(final String name, final Boolean stdDropTableRestrict) throws SQLException {
+        String url = getURL(name, true);
+        if (stdDropTableRestrict != null) {
+            url = addOption(url, "STANDARD_DROP_TABLE_RESTRICT", stdDropTableRestrict.toString());
+        }
+        return getConnectionInternal(url, getUser(), getPassword());
     }
```

Comment:
there is no need for such a method, just call 
    getConnection("cases;STANDARD_DROP_TABLE_RESTRICT=" + stdDropTableRestrict);